.eslintignore

```diff
@@ -1,5 +0,0 @@
-*.config.ts
-*.d.ts
-/src/graphql/data/__generated__/types-and-hooks.ts
-/src/graphql/thegraph/__generated__/types-and-hooks.ts
-/src/schema/schema.graphql
```

Comment:
noticed these three files aren't mentioned elsewhere but CI is passing just fine--were these ignores just unnecessary? (it makes sense to me that generated files *should* be ignored.

Reply:
I think this was just a very stale file. By using .eslintignore, we were actually typechecking all of the build files. Instead, we should have been using .gitignore.